src/browsergym/workarena/api/utils.py

```diff
@@ -172,3 +172,16 @@ def db_delete_from_table(instance: SNowInstance, sys_id: str, table: str) -> Non
 
     # Check for HTTP code 200 (fail otherwise)
     response.raise_for_status()
+
+
+def get_instance_sys_property(instance: SNowInstance, property_name: str) -> str:
```

Comment:
Why create a new function instead of just copying this one to utils.py?

https://github.com/ServiceNow/WorkArena/blob/8b9bb5bee1dee42275a271431a95aca88e466e47/src/browsergym/workarena/install.py#L85

Reply:
Also, I think /api/instance.py would be a better fit for this function since it's an instance specific API function

Reply:
Oh I didn't know we had the function already in utils. Let me do that